src/remote/activitypub/audience.ts

```diff
@@ -86,8 +86,7 @@ function isPublic(id: string) {
 }
 
 function isFollowers(id: string, actor: IRemoteUser) {
-	return [
-		`${actor.uri}/followers`,
-		// actor.followerUri,	// TODO
-	].includes(id);
+	return (
+		id == (actor.followersUri ? actor.followersUri : `${actor.uri}/followers`)
+	);
```

Comment:
Please use `===`
And, maybe below is enough.
```suggestion
  return id === actor.followersUri || `${actor.uri}/followers`;
```

Reply:
`||` will give a result of either True or False, not the URI, but I will use `===`

Reply:
> `||` will give a result of either True or False

That's no problem
![image](https://user-images.githubusercontent.com/30769358/105610904-82eef080-5df5-11eb-95b9-e5d0163e408c.png)

Reply:
Perhaps its specification is only equality operators.
![image](https://user-images.githubusercontent.com/30769358/105611188-1e349580-5df7-11eb-8760-aecf7da27a80.png)

Reply:
oh! I learned something, thank you!